docs/docs/icicle/arch_overview.md

```diff
@@ -1,34 +1,28 @@
 # Architecture Overview
 
-## Introduction
 
-ICICLE V3 is designed with flexibility and extensibility in mind, offering a robust framework that supports multiple compute backends and accommodates various cryptographic needs. This section provides an overview of ICICLE's architecture, highlighting its open and closed components, multi-device support, and extensibility.
+ICICLE v3 is designed with flexibility and extensibility in mind, offering a robust framework that supports multiple compute backends and accommodates various cryptographic needs. This section provides an overview of ICICLE's architecture, highlighting its open and closed components, multi-device support, and extensibility.
 
-## Open Frontend and CPU Backend
+### Open Frontend and CPU Backend
```

Comment:
I would remove the "Open" from the header here if we aren't having "Closed" below on CUDA backend.

Reply:
fixed